charts/newrelic-infra-operator/templates/_helpers.tpl

```diff
@@ -133,3 +133,16 @@ Return the customSecretLicenseKey
     {{- .Values.customSecretLicenseKey | default "" -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Returns fargate
+*/}}
+{{- define "newrelic.fargate" -}}
+{{- if .Values.global }}
```

Comment:
I'd love to simplify this, but it seems surprisingly complicated then :smile:

Reply:
I copied it from a different chart 😅